Gems/AtomLyIntegration/AtomBridge/Code/Source/AtomDebugDisplayViewportInterface.cpp

```diff
@@ -677,6 +679,30 @@ namespace AZ::AtomBridge
         }
     }
 
+    void AtomDebugDisplayViewportInterface::DrawPolyLine(AZStd::span<const AZ::Vector3> points, bool cycled)
+    {
+        if (m_auxGeomPtr)
+        {
+            AZStd::vector<AZ::Vector3> wsPoints;
+            wsPoints.resize_no_construct(points.size());
+            AZStd::transform(points.begin(), points.end(), wsPoints.begin(), [&](auto& pnt) {
+                return ToWorldSpacePosition(pnt);
+            });
+            AZ::RPI::AuxGeomDraw::PolylineEnd polylineEnd = cycled ? AZ::RPI::AuxGeomDraw::PolylineEnd::Closed : AZ::RPI::AuxGeomDraw::PolylineEnd::Open;
+            AZ::RPI::AuxGeomDraw::AuxGeomDynamicDrawArguments drawArgs;
+            drawArgs.m_verts = wsPoints.data();
+            drawArgs.m_vertCount = wsPoints.size();
+            drawArgs.m_colors = &m_rendState.m_color;
+            drawArgs.m_colorCount = 1;
+            drawArgs.m_size = m_rendState.m_lineWidth;
+            drawArgs.m_opacityType = m_rendState.m_opacityType;
+            drawArgs.m_depthTest = m_rendState.m_depthTest;
+            drawArgs.m_depthWrite = m_rendState.m_depthWrite;
+            drawArgs.m_viewProjectionOverrideIndex = m_rendState.m_viewProjOverrideIndex;
+            m_auxGeomPtr->DrawPolylines(drawArgs, polylineEnd);
+        }
+    }
+
     void AtomDebugDisplayViewportInterface::DrawPolyLine(const AZ::Vector3* pnts, int numPoints, bool cycled)
     {
         if (m_auxGeomPtr)
```

Comment:
Suggestion: Change this to call the other implementation so that we don't maintain two similar implementations.
```
    void AtomDebugDisplayViewportInterface::DrawPolyLine(const AZ::Vector3* pnts, int numPoints, bool cycled)
    {
        DrawPolyLine(AZStd::span<const AZ::Vector3>(pnts, numPoints), cycled);
    }
```